src/openloco/ui/WindowManager.cpp

```diff
@@ -1604,7 +1604,7 @@ namespace openloco::ui::WindowManager
             if (viewport->zoom != 0)
                 continue;
 
-            if (!viewport->contains({ position.x, position.y }))
+            if (!viewport->containsUI({ position.x, position.y }))
```

Comment:
This change is not correct. The function is checking a `viewport_pos` not a `xy32` (mouse coordinate)

Reply:
Incidently after changing it back to `contains` there is no need for the constructor and the value can be passed directly.